data/macos.yaml

```diff
@@ -314,7 +314,7 @@ urls:
 - 'https://forensicswiki.xyz/wiki/index.php?title=Mac_OS_X'
 - 'https://forensicswiki.xyz/wiki/index.php?title=Mac_OS_X_10.9_-_Artifacts_Location#Kernel_Extension'
 ---
-name: MacOSDuetKnowledgeBase
+name: MacOSDuetKnowledge Base
```

Comment:
Please change this back, looks like one too many find replace ;)

Reply:
Haha, my `sed` command was too effective ;)